metpy/io/tests/test_pandas_to_netcdf.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2019 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Test the `pandas_to_netcdf` module."""
+
+import logging
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+import xarray as xr
+
+from metpy.cbook import get_test_data
+from metpy.io import dataframe_to_netcdf
+
+# Turn off the warnings for tests
+logging.getLogger('metpy.io.pandas_to_netcdf').setLevel(logging.CRITICAL)
+
+
+def test_dataframe_to_netcdf_basic(tmpdir):
+    """Test dataframe conversion to netcdf."""
+    df = pd.read_csv(get_test_data('parsed_metars_example3.csv'))
```

Comment:
Would it be possible to use the existing `station-data.txt` in `staticdata` rather than adding `parsed_metars_example3.csv`?

Reply:
With a little wrangling for readability, this was doable. I dropped the added data file.